tareas/TareasSarahi/Tarea1TSFCII-1.jl

```diff
@@ -0,0 +1,76 @@
+#using Pkg; Pkg.activate("../")
+#
+#Pkg.instantiate()
+
+#function Newton(f,df,x0,N) #Definimos el método de Newton (necesita una función, su derivada, un x0 inicial y el número de #iteraciones)
+#    c=0 #iniciamos un contador en cero 
+#    x1=0.0
+#    while c<N #si el contador es menor que el núemro de pasos deaseado, el ciclo continua
+#        x1=x0-(f(x0)/df(x0)) #Newton
+#        x0=x1 #renombramos la x0 inicial
+#        c=c+1 #aumentamos en uno el contador a cada ciclo realizado
+#    end
+#    return x1 #devolvemos la raíz 
+#end
```

Comment:
Lo que haces en esta función es correcto; te puedes "ahorrar" el contador si en lugar del un ciclo `while` usas un ciclo `for` en el rango `1:N`.

En cuanto al comentario que detalla la función, escríbelo *antes* de la función (sin usar espacio después del primer `#` para que sea un comentario para Julia.